packages/pci-sdk-iframe/src/apiClient/index.ts

```diff
@@ -23,12 +24,14 @@ export async function request2FACode(): Promise<IRequest2FACodeResponse> {
 		headers,
 		body: JSON.stringify({ show_verification_secret: true }),
 	})
-		.then((res) => {
+		.then(async (res) => {
```

Comment:
I think the easiest method would be:

```ts
case: 401:
  const {code} = await res.json();
  const errorMessage = errorsService. get401ResponseMessage(code);
  throw new Error(errorMessage);
```

Reply:
also we could refactor this a little bit and parse the json just one time

Reply:
@IagoLast This was exactly what I tried at first but Eslint won't let you put const, let or var in a switch block.